examples/analytics.amp.html

```diff
@@ -22,9 +22,9 @@
 <amp-analytics id="analytics1">
 <script type="application/json">
 {
-  "host": "example.com",
+  "transport": {"beacon": false, "xhrpost": false},
```

Comment:
Is the intention that "image" is enabled by default? From what I can tell in the code, this is not the case...